libs/ngx-dsfr/alert/src/lib/alert.component.ts

```diff
@@ -39,25 +39,37 @@ export class DsfrAlertComponent implements OnInit, AfterViewInit {
 
   ngOnInit(): void {
 
-    if (!this.type) {
+    if ( !this.type ) {
       throw EMPTY_TYPE_ERROR;
     }
 
     this.initClasses();
   }
```

Comment:
Given what's in `ngOnChanges` and the fact that it is called once **before** `ngOnInit`, I think you can remove it ( _and the `OnInit` interface, and its import_ )